recipes/dram/meta.yaml

```diff
@@ -21,16 +21,16 @@ requirements:
     - python >=3.6
     - pip
   run:
-    - python >=3.6, <3.11
+    - python =3.11
```

Comment:
why is this strict pin needed?

Reply:
it may not be needed, but scikit-bio 5.8 requires 3.11 anyway.